### PR TITLE
Refactoring Website

### PR DESCRIPTION
**Pull request by ghost**
I'm refactoring some codes without changing much of the layout.

Here is what I have done:

- Use semantic HTML
- Add `description` meta for SEO
- Use `class` instead of `id` for styling
- Delete the `app.js` and move the script to `index.html` file because it's only have a few code
- Remove the `snap-scrolling` style because can overflow the scrollbar
- Self-host the Roboto fonts for a better performance
- Compress some images

Here is the difference from the score:

**Before:**
firefox_MtnoYCcWsV

**After:**
firefox_9sueLwrQkf

Here is the demo of my changes: https://wahyuwiyoko.github.io/libre-tube.github.io/

Better to test it on desktop and mobile before merge this PR.
I hope I don't break up the site layout :grin:

### CODE REVIEW

**Comment by ghost** [comment]
Oh and one thing, I changed the `assets` directory name to `images`. So I saw there's [LibreTube README](https://github.com/libre-tube/LibreTube/blob/master/README.md?plain=1) are using that URL. I will make PR to fix the README if this merged.

**Comment by Bnyro** [comment]
Hey, thank you for your interest in contributing and your work concerning the website!

I just tested your demo and would suggest some minor changes:

image

1. Previously, the "Privacy Simplified" `h1` has been center aligned while the social links and donation button have been aligned to the bottom. I would suggest to keep it like that, it feels pretty much squeezed when being centered completely.

image

image

2. I'm still a fan of the top bar padding we had here, I would suggest to increase the header padding again to make it feel less tight.

image

3. The width of the articles in the FAQ section could be decreased/their margin could be increased in my opinion, it just looks really stretched on larger screens now since the questions are really short.

Regards and thanks again for your work!

**Comment by Bnyro** [comment]
And one last thing I forgot to mention:
What exactly was the issue with snap-scrolling?
It got added by another PR and I'm pretty happy with it, I don't think we need to remove it :thinking:

**Comment by ghost** [comment]
> 1. Previously, the "Privacy Simplified" `h1` has been center aligned while 
> the social links and donation button have been aligned to the bottom. 
> I would suggest to keep it like that, it feels pretty much squeezed when being centered completely.
>  2. I would suggest to increase the header padding again to make it feel less tight.

I'm increased the `padding-top` of the `#about` style (the container) to `6rem` so it's goes a bit to bottom. And the `header` padding to `2rem` (top and bottom) and `5rem` (left and right). Here is how it's look like:
firefox_mz7my5WaZj

>  3. The width of the articles in the FAQ section could be decreased/their margin 
>  could be increased in my opinion, it just looks really stretched on larger screens now since the questions are really short.

I changed the width of `.faq` to 60%. Here is how it's look like:
firefox_OZmDKnLW1Q

~~What exactly was the issue with snap-scrolling?~~

~~I had problem with that because the `main` and `footer` are separated that caused the scrollbar overflow. Only the `main` had `snap-scrolling` class. It will make 2 vertical scrollbar and the `footer` behavior looks like `fixed` position if I scrolled.~~
 
I have added the snap-scrolling back now 😄 
Check the new changes on the demo: https://wahyuwiyoko.github.io/libre-tube.github.io/

File: style.css [hunk 1]
```diff
@@ -13,151 +43,147 @@ html {
...
+  left: 0;
+  right: 0;
   background-color: var(--background);
-  padding: 0 5vw;
-  padding-top: 5vh;
-  padding-bottom: 2vh;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  padding: 1.5rem 2.5rem 1.5rem 2rem;
```
Comment (wahyuwiyoko) [inline style.css:70]: Tried change to `padding: 2rem 5rem;`

File: style.css [hunk 2]
```diff
@@ -13,151 +43,147 @@ html {
...
-  justify-content: space-between;
+.download-links img {
+  width: 15rem;
 }
 
-#slogan {
-  margin-top: 42vh;
+/* About Section */
+#about {
+  padding-top: 1rem;
```
Comment (wahyuwiyoko) [inline style.css:112]: The `Privacy Simplified` container goes a bit to bottom with `padding-top: 4rem;`

File: style.css [hunk 3]
```diff
@@ -13,151 +43,147 @@ html {
...
-  margin: 0.5em 5%;
+.highlights h2 {
+  font-size: 1.4rem;
 }
 
-details {
-  margin: 0.7em 0;
+/* FAQ Section */
+.faq {
+  width: 90%;
```
Comment (wahyuwiyoko) [inline style.css:182]: Decreased the `.faq` container width to 60%

**Comment by Bnyro** [comment]
Tested, works well now, thank you for your help!

**Pull request merged**